openapi/nytimes.moviereviews/types.bal

```diff
@@ -29,10 +29,10 @@ public type Movie record {
 };
 
 public type MovieMultimediaResource record {
-    int height?;
-    string src?;
-    string 'type?;
-    int width?;
+    int? height?;
```

Comment:
Fields are not originally nillable. Please regenerate without giving `--nullable` in the command.